assets/js/components/GlobalFilters.vue

```diff
@@ -183,15 +179,35 @@ export default Vue.extend({
           }
         }
       }
+
       return filters;
     },
   },
 
+  created() {
+    const self = this;
+    this.chronologicalFilters = [];
+
+    this.makeChronologicalFiltersFromExisting();
+    this.handleEsc();
+  },
 
   methods: {
-    removeFilter(e){
-      const remove_el = e.target.getAttribute('filter')
-      this.filters[remove_el] = null;
+    makeChronologicalFiltersFromExisting() {
+      for(var i in this.filters) {
+        if(this.filters[i]) {
+          this.chronologicalFilters.push({type: i, val: this.filters[i]});
```

Comment:
why do you care about the value? did you want to build a full undo system?

Reply:
you are right, they are not needed, used them only for debugging. will remove